sdk/core/Azure.Core/src/Diagnostics/AzureCoreEventSource.cs

```diff
@@ -30,94 +31,94 @@ internal sealed class AzureCoreEventSource : EventSource
 
         public static AzureCoreEventSource Singleton { get; } = new AzureCoreEventSource();
 
-        [Event(RequestEvent, Level = EventLevel.Informational)]
+        [Event(RequestEvent, Level = EventLevel.Informational, Message = "Request [{0}] {1} {2} \r\n{3}")]
         public void Request(string requestId, string method, string uri, string headers)
         {
             WriteEvent(RequestEvent, requestId, method, uri, headers);
         }
 
-        [Event(RequestContentEvent, Level = EventLevel.Verbose)]
+        [Event(RequestContentEvent, Level = EventLevel.Verbose, Message = "Request [{0}] Content {1}")]
```

Comment:
Is it worth logging `content` in this case? It likely just to come out as `System.Byte[]`.

Reply:
It won't, we have a special formatter that know about `byte[]` https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/core/Azure.Core/src/Shared/EventSourceEventFormatting.cs#L13